src/CodeActionOnSave.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  CodeAction,
+  CodeActionKind,
+  type CodeActionProvider,
+  type ProviderResult,
+} from "vscode";
+
+const FORCE_FORMAT_DOCUMENT_COMMAND = "prettier.forceFormatDocument";
+const FORCE_FORMAT_DOCUMENT_CODE_ACTION_KIND = CodeActionKind.Source.append(
+  FORCE_FORMAT_DOCUMENT_COMMAND
+);
+const FORCE_FORMAT_DOCUMENT_TITLE = "Format Document (Forced)";
```

Comment:
Why is it forced?

Reply:
Because this code action uses a command of the same name that exists in the command palette.
But I would be willing to adopt any better name. 👍